### ISSUE TITLE
Cannot resolve com.amazon.opendistroforelasticsearch:opendistro_security:6.8.3-0.10.1.3

### ISSUE THREAD

**Issue by larsw** [ThalesGroup/reflex-elasticsearch-ranger#6]
Hi and greetings from Thales Norway,

I'm trying to build the project by following the instruction in the README, but it can't find one of the dependencies.

Tried to change version of the aforementioned dependency to `1.9.0.0` (latest I found here: https://search.maven.org/search?q=com.amazon.opendistroforelasticsearch) but then it complains about some breaking API changes  during compile.

```shell
[ERROR] /home/larsw/code/reflex-elasticsearch-ranger/ranger-privileges-evaluator/src/main/java/com/guavus/ranger/RangerPrivilegesEvaluator.java:[20,68] cannot find symbol
[ERROR]   symbol:   class ActionGroupHolder
[ERROR]   location: package com.amazon.opendistroforelasticsearch.security.configuration
[ERROR] /home/larsw/code/reflex-elasticsearch-ranger/ranger-privileges-evaluator/src/main/java/com/guavus/ranger/RangerPrivilegesEvaluator.java:[94,48] cannot find symbol
[ERROR]   symbol: class AbstractPrivilegesEvaluator
[ERROR] /home/larsw/code/reflex-elasticsearch-ranger/ranger-privileges-evaluator/src/main/java/com/guavus/ranger/RangerPrivilegesEvaluator.java:[141,99] cannot find symbol
[ERROR]   symbol:   class ActionGroupHolder
[ERROR]   location: class com.guavus.ranger.RangerPrivilegesEvaluator
```

**Comment by divyansh-jain-guavus** [comment]
Hi

1. Unzip and place the attached jar to your local maven repo on the path ~/.m2/repository/org/apache/hadoop/apache-hadoop-kerberos-util/1.0.0/

[apache-hadoop-kerberos-util-1.0.0.jar.zip](https://github.com/ThalesGroup/reflex-elasticsearch-ranger/files/5298296/apache-hadoop-kerberos-util-1.0.0.jar.zip)

2. Clone and build https://github.com/ThalesGroup/opendistro-security/tree/opendistro-0.10
using the commands provided here -> https://github.com/ThalesGroup/opendistro-security#test-and-build

These are needed for ranger-privileges-evaluator as it works as an extension of opendistro-security

3. Next you should change the version in the file https://github.com/ThalesGroup/reflex-elasticsearch-ranger/blob/master/ranger-privileges-evaluator/pom.xml

<opendistro.version>0.10.1.3</opendistro.version> change the value to->  0.10.1.4

4. Now build the project again

**Comment by goelrajat** [comment]
Hi Divynash,

Instead of step 1 'Unzip and place attached jar and pom to local maven' for Hadoop kerberos util, we should add a step to clone that repo and build it using maven. This will ensure jar gets deployed to local maven. Finally, we can build reflex-elasticsearch-ranger repo.

**Comment by divyansh-jain-guavus** [comment]
The Hadoop kerberos util repo is currently private or else I would have added those steps.

**Comment by larsw** [comment]
Uhm, ok... any plans on releasing all the internal dependencies properly in the Maven Central Registry?

**Comment by goelrajat** [comment]
Yes, the plan to move Hadoop Kerberos util dependency as a public repository and release it in Maven Central Registry is in progress. Will update when its done.

**Comment by larsw** [comment]
Excellent.
